pynitrokey/cli/storage.py

```diff
@@ -49,5 +49,120 @@ def enable_update(password):
         local_print("No Nitrokey Storage device found")
 
 
+@click.command()
+@click.option(
+    "-p",
+    "--password",
+    prompt="User PIN required",
+    hide_input=True,
+    help="User password",
+)
+def open(password):
+    """Unlock the encrypted storage"""
+    try:
+        nks = NitrokeyStorage()
+        nks.connect()
+        ret = nks.unlock_encrypted_volume(password)
+        if not ret.ok:
+            if ret == RetCode.WRONG_PASSWORD:
+                local_print("Wrong user PIN")
+            else:
+                local_print("Error unlocking the encrypted storage")
+    except DeviceNotFound:
+        local_print("No Nitrokey Storage device found")
+
+
+@click.command()
+def close():
+    """Lock the encrypted storage"""
+    try:
+        nks = NitrokeyStorage()
+        nks.connect()
+        nks.lock_encrypted_volume()
+    except DeviceNotFound:
+        local_print("No Nitrokey Storage device found")
+
+
+@click.command()
+@click.option(
+    "-p",
+    "--password",
+    prompt="Please input the hidden storage passphrase",
+    hide_input=True,
+    help="Hidden storage passphrase",
+)
+def open_hidden(password):
+    """Unlock an hidden volume"""
+    try:
+        nks = NitrokeyStorage()
+        nks.connect()
+        ret = nks.unlock_hidden_volume(password)
+        if not ret.ok:
+            if ret == RetCode.WRONG_PASSWORD:
+                local_print("Wrong hidden storage passphrase")
+            else:
+                local_print("Error unlocking the hidden storage")
+    except DeviceNotFound:
+        local_print("No Nitrokey Storage device found")
+
+
+@click.command()
+def close_hidden():
+    """Lock the hidden volumes"""
+    try:
+        nks = NitrokeyStorage()
+        nks.connect()
+        nks.lock_hidden_volume()
+    except DeviceNotFound:
+        local_print("No Nitrokey Storage device found")
+
+
+@click.command()
+@click.argument(
+    "slot",
+    type=int,
+)
+@click.argument(
+    "begin",
+    type=int,
+)
+@click.argument("end", type=int)
+@click.option(
+    "-p",
+    "--password",
+    prompt="Please input the hidden storage passphrase",
+    hide_input=True,
+    help="Hidden storage passphrase",
+)
+def create_hidden(slot, begin, end, password):
+    """Create an hidden volume
+
+    SLOT is the slot used for the hidden volume (0-3)\n
+    START is where the volume begins expressed in percent of total available storage (0-99)\n
+    END is where the volume ends expressed in percent of total available storage (1-100)\n
+    """
+    if not slot in [0, 1, 2, 3]:
+        local_print("Slot must be between 0 and 3")
+        exit(1)
```

Comment:
Please raise a `CliException` or use `local_critical` instead of calling `exit`.  ([`exit`](https://docs.python.org/3/library/constants.html#exit) should only be used in the interpreter, not in programs.)